### PR TITLE
tests: start a few unit tests

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

File: build/test.mk [hunk 1]
```diff
@@ -22,7 +22,11 @@ GOTOOLS += github.com/stretchr/testify/assert
 test: test-only
 test-only: test-unit test-integration
 
-test-unit:
+test-prep: compile-only
```
Comment (sanderblue) [inline build/test.mk:25]: Nice 👍

File: internal/schema/schema_util_test.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		"String",
+	}
+
+	for _, r := range *results {
+		hasString := stringInStrings(r.Name, expectedNames)
+		assert.True(t, hasString)
+	}
+}
+
+func stringInStrings(s string, ss []string) bool {
```
Comment (sanderblue) [inline internal/schema/schema_util_test.go:46]: 🐍

File: testdata/goodConfig_fixture.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+log_level: trace
+cache:
+  schema_file: testingtesting.schema.json
```
Comment (sanderblue) [inline testdata/goodConfig_fixture.yml:4]: Intentional `testingtesting`?

File: testdata/goodConfig_fixture.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+endpoint: https://api-staging.newrelic.com/graphql
```
Comment (sanderblue) [inline testdata/goodConfig_fixture.yml:5]: Does this have to be staging?

File: testdata/goodConfig_fixture.yml (hunk 3 above)
Comment (zlesnr) [reply testdata/goodConfig_fixture.yml:4]: Oh yeah, but I can clean that up.

File: testdata/goodConfig_fixture.yml (hunk 4 above)
Comment (zlesnr) [reply testdata/goodConfig_fixture.yml:5]: No, I just want it to be non-default, so we know its being loaded from this config.

File: internal/schema/schema_util_test.go (hunk 2 above)
Comment (zlesnr) [reply internal/schema/schema_util_test.go:46]: :sheep: ish...

File: testdata/goodConfig_fixture.yml (hunk 3 above)
Comment (zlesnr) [reply testdata/goodConfig_fixture.yml]: Updated.

File: testdata/goodConfig_fixture.yml (hunk 4 above)
Comment (zlesnr) [reply testdata/goodConfig_fixture.yml]: Updated.

**Pull request merged**